homeassistant/components/sensor/jewish_calendar.py

```diff
@@ -0,0 +1,141 @@
+"""
+Platform to retrieve Jewish calendar information for Home Assistant.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.jewish_calendar/
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_LATITUDE, CONF_LONGITUDE, CONF_NAME
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+import homeassistant.util.dt as dt_util
+
+
+REQUIREMENTS = ['hdate==0.6.3']
+
+_LOGGER = logging.getLogger(__name__)
+
+SENSOR_TYPES = {
+    'date': ['Date', 'mdi:judaism'],
+    'weekly_portion': ['Parshat Hashavua', 'mdi:book-open-variant'],
+    'holiday_name': ['Holiday', 'mdi:calendar-star'],
+    'holyness': ['Holyness', 'mdi:counter'],
+    'first_light': ['Alot Hashachar', 'mdi:weather-sunset-up'],
+    'gra_end_shma': ['Latest time for Shm"a GR"A', 'mdi:calendar-clock'],
+    'mga_end_shma': ['Latest time for Shm"a MG"A', 'mdi:calendar-clock'],
+    'plag_mincha': ['Plag Hamincha', 'mdi:weather-sunset-down'],
+    'first_stars': ['T\'set Hakochavim', 'mdi:weather-night'],
+}
+
+CONF_DIASPORA = 'diaspora'
+CONF_LANGUAGE = 'language'
+CONF_SENSORS = 'sensors'
+
+DEFAULT_NAME = 'Jewish Calendar'
+DEFAULT_LATITUDE = 31.778
```

Comment:
We should default to the users location

Reply:
Oh, you already do that. In that case, please remove these as they serve no purpose.